sktime/forecasting/base/adapters/_darts.py

```diff
@@ -0,0 +1,377 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements adapter for Darts models."""
+import abc
+from typing import List, Optional
+
+import pandas as pd
+
+from sktime.forecasting.base import BaseForecaster, ForecastingHorizon
+
+__author__ = ["yarnabrina", "fnhirwa"]
+
+
+class _DartsAdapter(BaseForecaster):
+    """Base adapter class for Darts models.
+
+    Parameters
+    ----------
+    past_covariates : Optional[List[str]], optional
+        column names in ``X`` which are known only for historical data, by default None
+    num_samples : Optional[int], optional
+        Number of times a prediction is sampled from a probabilistic model, by default
+        1000
+
+    Notes
+    -----
+    If unspecified, all columns will be assumed to be known during prediction duration.
+    """
+
+    _tags = {
+        # packaging info
+        # --------------
+        "authors": ["yarnabrina", "fnhirwa"],
+        "maintainers": ["yarnabrina", "fnhirwa"],
+        "python_version": ">=3.9",
+        "python_dependencies": ["u8darts"],
+        "python_dependencies_alias": {"u8darts": "darts"},
+        # estimator type
+        # --------------
+        "y_inner_mtype": "pd.DataFrame",
+        "X_inner_mtype": "pd.DataFrame",
+        "requires-fh-in-fit": False,
+        "enforce_index_type": pd.DatetimeIndex,
+        "handles-missing-data": False,
+        "capability:pred_int": False,
```

Comment:
L270-321 is defining `predict_quantiles`. Why is it `False` then?

Reply:
Also, question for @fkiraly, I think we have a check in testing that value of this tag is being checked against presence of either predict_quantiles or predict_interval. How is that test passing now, as CI shows all green?

Reply:
This was an overlook let me change it to `True`